bindings/wallet-c/src/lib.rs

```diff
@@ -413,6 +493,23 @@ pub unsafe extern "C" fn iohk_jormungandr_wallet_delete_string(ptr: *mut c_char)
     }
 }
 
+/// Delete a binery buffer that was returned by this library alongside with its
+/// length.
+///
+/// # Safety
+///
+/// This function dereference raw pointers. Even though
+/// the function checks if the pointers are null. Mind not to put random values
+/// in or you may see unexpected behaviors
+#[no_mangle]
+pub unsafe extern "C" fn iohk_jormungandr_waller_delete_buffer(ptr: *mut c_char, length: usize) {
+    if !ptr.is_null() {
+        let data = std::slice::from_raw_parts_mut(ptr, length);
+        let data = Box::from_raw(data as *mut [c_char]);
+        std::mem::drop(data);
+    }
```

Comment:
is there any specific reason for `c_char` here instead of just `u8`?

Reply:
Nice point. `u8` makes much more sense actually.

Reply:
Pushed an update, please make sure it doesn't break your code.